views/js/pciManager/pciManager.js

```diff
@@ -115,68 +116,106 @@ define([
+                            off: {
+                                active: $li.hasClass('pci-disabled')
                             }
-                        }).on('on', function(){
+                        }).on('on', function () {
                             $li.removeClass('pci-disabled');
-                            $.getJSON(urls.enableUrl, {typeIdentifier : typeIdentifier}, function(data){
-                                if(data.success){
+                            $.getJSON(urls.enableUrl, {typeIdentifier: typeIdentifier}, function (data) {
+                                if (data.success) {
                                     listing[typeIdentifier].enabled = true;
                                     self.trigger('pciEnabled', typeIdentifier);
                                 }
                             });
-                        }).on('off', function(){
+                        }).on('off', function () {
                             $li.addClass('pci-disabled');
-                            $.getJSON(urls.disableUrl, {typeIdentifier : typeIdentifier}, function(data){
-                                if(data.success){
+                            $.getJSON(urls.disableUrl, {typeIdentifier: typeIdentifier}, function (data) {
+                                if (data.success) {
                                     listing[typeIdentifier].enabled = false;
                                     self.trigger('pciDisabled', typeIdentifier);
                                 }
                             });
                         });
                     });
 
-                    hider.show($fileContainer);
-
-                }else{
+                    $fileContainer.find('.actions').each(function () {
+                        var $unregister = $(this).find('.unregister');
+                        var $listElem = $(this).closest('.pci-list-element');
+                        var typeIdentifier = $listElem.data('type-identifier');
+                        buttonFactory({
+                            id: 'unregister',
+                            type: 'info',
+                            icon: 'bin',
+                            label: 'Delete',
+                            renderTo: $unregister
+                        }).on('click', function () {
+                            confirmBox(
+                                __('You are about to delete the portable Custom Interaction %s from the system. ', typeIdentifier),
+                                function () {
+                                    $.getJSON(urls.unregisterUrl, {typeIdentifier: typeIdentifier}, function (data) {
+                                        if (data.success) {
+                                            delete listing[typeIdentifier];
+                                            self.trigger('pciDisabled', typeIdentifier);
+                                        } else {
+                                            console.log(data);
```

Comment:
Please remove `console.log` statements. We only allow it in exceptional circumstances.